src/main.tf

```diff
@@ -86,6 +107,16 @@ resource "aws_launch_template" "nodes" {
 
   update_default_version = true
 
+  image_id = local.gpu_enabled_instance_types[each.key] ? data.aws_ssm_parameter.eks_gpu_ami[0].value : data.aws_ssm_parameter.eks_ami.value
+
+  user_data = base64encode(
+    <<EOF
+#!/bin/bash
+set -o xtrace
+/etc/eks/bootstrap.sh ${local.cluster_name} --kubelet-extra-args '--node-labels=node.kubernetes.io/instancegroup=${each.key}'
```

Comment:
Is anything else needed in this file? Did you check to see what this file looked like on a default node before this change?

Reply:
I did look and it's a massive file. I'll paste the contents of it in here after deploying a main branch cluster.

Reply:
[bootstrap.txt](https://github.com/massdriver-cloud/aws-eks-cluster/files/15503769/bootstrap.txt)